plugins/permission-backend-module-opa-wrapper/README.md

```diff
@@ -8,15 +8,20 @@ This wrapper is still in **development**, you can use it at your own risk, be aw
 
 ## Pre-requisites
 
-- This plugin also requires and assumes that you have set up and followed the instructions in the [Backstage Permissions Docs](https://backstage.io/docs/permissions/overview) as it of course relies on the permissions framework to be there and set up.
+- This plugin requires that you have set up and followed the instructions in the [Backstage Permissions Docs](https://backstage.io/docs/permissions/overview) as it relies on the Backstage permissions framework
+- The [backstage-opa-backend](../backstage-opa-backend) module
```

Comment:
This shouldn't be the case anymore, but if so please LMK!

You should be able to use it as is without the backend plugin now.